### DISCUSSION TITLE
Actions for moving blocks (copy and cut, detach, insert, etc)

### DISCUSSION THREAD

**Discussion by BeksOmega** [RaspberryPiFoundation/blockly-keyboard-experimentation#19] [General]
How might we enable users to move blocks around the workspace using keyboard navigation?

There are several different actions we could provide to users to allow them to move blocks around, which set makes the most sense?
1) Copy
2) Cut
3) Paste
4) Mark (and unmark)
5) Insert
6) Detach

(This list is inexhaustive, but should be a good start!)

**Comment by BeksOmega** [comment]
My personal opinion is that copy, cut, and paste are standard actions, and that they're sufficient to allow folks to do anything they want to do.

**Reply by cassandralfernandes** [reply]
I agree that copy, cut, and paste are essential actions and feel like the most intuitive starting point. I wonder how challenging it will be for (non-sighted and sighted) users to get the paste location to be where they intend it to be. I'm curious to see how that looks when testing with users
